libr/config.h.in

```diff
@@ -8,17 +8,19 @@
 #define R_INLINE 0
 #define R2_LOADLIBS 1
 
-#define R_EGG_STATIC_PLUGINS @plugins_egg@
-#define R_BIN_STATIC_PLUGINS @plugins_bin@
-#define R_FS_STATIC_PLUGINS @plugins_fs@
-#define R_IO_STATIC_PLUGINS @plugins_io@
-#define R_DEBUG_STATIC_PLUGINS @plugins_debug@
-#define R_BP_STATIC_PLUGINS @plugins_bp@
-#define R_ASM_STATIC_PLUGINS @plugins_asm@
-#define R_ANAL_STATIC_PLUGINS @plugins_anal@
-#define R_CORE_STATIC_PLUGINS @plugins_core@
-#define R_LANG_STATIC_PLUGINS @plugins_lang@
-#define R_PARSE_STATIC_PLUGINS @plugins_parse@
-#define R_CRYPTO_STATIC_PLUGINS @plugins_crypto@
+#define R_EGG_STATIC_PLUGINS @plugins_egg@, NULL
```

Comment:
Wtf

Reply:
I just replaced trailing `0` with `NULL` and moved it to `config.h.in`: https://github.com/radare/radare2/blob/master/meson.build#L157

Reply:
I can bring it back to `meson.build` if required.

Reply:
Looks a bit ugly to me . I prefer to keep those definitions without ending null, because is the implementation wich refers to those defines the one that ensures that its null terminated

Reply:
Sorry, I didn't understand. Should I move trailing `NULL` to `meson.build` or remove it totally?
Configure adds trailing `0` to this macros: https://github.com/radare/radare2/blob/master/configure-plugins#L75
Additional trailing NULL: https://github.com/radare/radare2/blob/master/libr/bin/bin.c#L31